frontend/src/services/FeatureFlagManager.ts

```diff
@@ -116,25 +122,25 @@ export class FeatureFlagsManager {
     );
   }
 
+  /*
+
+    - Flags set by environment variables are the first override to default values
+    - Flags set in the /dev/feature-flags admin view will be set in the cookie
+    - Flags set in query params will result in the flag value being stored in the cookie
+    - As query param values are set in middleware on each request, query params have the highest precedence
+    - Values set in cookies will be persistent per browser session unless explicitly overwritten
+    - This means that simply removing a query param from a url will not revert the feature flag value to
+    the value specified in environment variable or default, you'll need to clear cookies or open a new private browser
+
+  */
   get featureFlags(): FeatureFlags {
     return {
       ...this.defaultFeatureFlags,
+      ...this.featureFlagsFromEnvironment,
```

Comment:
note that this means that if a user in any way sets their cookie to override the flag, they'll be able to access search or opportunities